gtk/Cargo.toml

```diff
@@ -7,7 +7,7 @@ name = "gtk"
 homepage = "https://gtk-rs.org/"
 authors = ["The gtk-rs Project Developers"]
 readme = "README.md"
-version = "0.16.0"
+version = "0.16.1"
```

Comment:
There are more crates in here. Best to update them all in sync (and only upload the ones that have actual changes). That's what we do elsewhere.

Reply:
Done!

Reply:
Not for the `sys` crates though, should I do it for them too?

Reply:
All

Reply:
Done as well!